DigitalLearningSolutions.Web/Views/Login/ChooseACentre.cshtml

```diff
@@ -15,65 +15,80 @@
 
     <h1 id="page-heading" class="nhsuk-heading-xl">Choose a centre</h1>
     <p class="nhsuk-body-l">
-      You are registered at more than one centre. Please choose the centre you would like to sign in to.
+      Please choose the centre you would like to sign in to.
     </p>
 
-    @* TODO HEEDLS-912: Remove this temporary button *@
-    <a role="button" class="nhsuk-button delete-button" style="font-size: xx-large; height: 100px; width: 100%;"
-       asp-action="ChooseCentre" asp-route-centreId="0">
-      Log into first available temporary
-    </a>
+    @if (Model.CentreUserDetails.Any(centreRow => centreRow.IsDelegateUnapproved)) {
+      <vc:inset-text css-class="" text="You have an unapproved delegate account. You will receive an email at the address you registered with when your registration has been approved." />
+    }
 
-    @* TODO HEEDLS-912: Add the Reset Password link and description *@
-    <table role="table" class="nhsuk-table-responsive">
-      <thead role="rowgroup" class="nhsuk-table__head">
-      <tr role="row">
-        <th role="columnheader" scope="col">
-          Centre
-        </th>
-        <th role="columnheader" scope="col">
-          Roles
-        </th>
-        <th role="columnheader" scope="col">
-          Action
-        </th>
-      </tr>
-      </thead>
-      <tbody class="nhsuk-table__body">
-      @foreach (var centreRow in Model.CentreUserDetails) {
-        <tr role="row" class="nhsuk-table__row">
-          <td role="cell" class="nhsuk-table__cell choose-centre-td">
-            <span class="nhsuk-table-responsive__heading">Centre: </span>
-            @centreRow.CentreName
-            @* TODO HEEDLS-912: Add tags for the new fields here *@
-          </td>
-          <td role="cell" class="nhsuk-table__cell choose-centre-td">
-            <span class="nhsuk-table-responsive__heading">Roles: </span>
-            <div class="centre-role-tags">
-              @if (centreRow.IsAdmin) {
-                <div>
+    @if (Model.CentreUserDetails.Count > 1) {

```

Comment:
Question I can't quite figure out. Are there any cases where this count will be exactly 1? I think if they have a single account and `LoginService.DetermineDestinationForSuccessfulLogin` returns us to `ChooseACentre` when logging into that single account because centre is inactive etc. then we won't be displaying their single account?

Reply:
Errrrrrm....yes, I think you are correct, but this actually looks like a dumb mistake by me, and for the sake of accuracy in the view it should be `> 0`. Fixed.